ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml

```diff
@@ -47,11 +44,16 @@ tag_specification:
 tests:
 - as: ocs-operator-ci
   commands: |
-    make ocs-operator-ci
+    GOPATH=$(mktemp -d) make ocs-operator-ci
```

Comment:
I don't think setting the gopath here should be necessary. For now I'm testing this out to try and better understand some permission errors that are occurring.